pkgs/racket-build-guide/contribute.scrbl

```diff
@@ -76,15 +76,30 @@ See the @secref["contribute-guidelines"].
 @section[#:tag "pkg-contribute"]{Distribution-Package Contributions}
 
 If you find yourself changing a file that is in a
-@filepath{share/pkgs} subdirectory, then that file is not part of the
-main Racket Git repository. It almost certainly has its own Git
-repository somewhere else, possibly within
+@filepath{share/pkgs} subdirectory (either installed as part of a
+Racket release or as a product of an in-place build), then that file
+is not part of the main Racket Git repository. It almost certainly has
+its own Git repository somewhere else, possibly within
 @url{https://github.com/racket}, but possibly in another user's space.
 The name of the directory in @filepath{share/pkgs} is almost certainly
 the package name.
 
-To start working on a package @nonterm{pkg-name}, it's usually best to
-go to the root directory of your Racket repository checkout and run
+To start working on a package @nonterm{pkg-name} from a Racket release
+or snapshot, you first need to edit the package's catalog
```

Comment:
"edit the package's catalog" is not quite right here. This should be something like "adjust the package installation to use the source specified by the main package catalog".

Reply:
Ah thanks, that's a better description of what's happening. 😄 I have updated this to match your wording.